src/Renderer/c3DEngine.js

```diff
@@ -28,6 +28,7 @@ function c3DEngine(scene, positionCamera, viewerDiv, debugMode, gLDebug) {
     this.viewerDiv = viewerDiv;
     this.debug = debugMode;
     this.scene3D = new THREE.Scene();
+    this.scene3D.autoUpdate = false;
```

Comment:
This change is unrelated to the current PR so I'd prefer that you submit a separate PR for this.